tests/test_library.py

```diff
@@ -300,6 +300,34 @@ def test_library_MovieSection_collection_exception(movies):
         movies.collection("Does Not Exists")
 
 
+@pytest.mark.authenticated
+def test_library_MovieSection_managedHubs(movies):
+    recommendations = movies.managedHubs()
+    with pytest.raises(BadRequest):
+        recommendations[0].remove()
+    first = recommendations[0]
+    first.promoteRecommended().promoteHome().prmoteShared().reload()
```

Comment:
Typo and remove reload.
```suggestion
    first.promoteRecommended().promoteHome().promoteShared()
```